ddtrace/contrib/grpc/client_interceptor.py

```diff
@@ -182,6 +183,12 @@ def _intercept_client_call(self, method_kind, client_call_details):
         span._set_str_tag(constants.GRPC_METHOD_KIND_KEY, method_kind)
         span._set_str_tag(constants.GRPC_HOST_KEY, self._host)
         span._set_str_tag(constants.GRPC_PORT_KEY, self._port)
+=======
+        set_grpc_method_meta(span, client_call_details.method, method_kind)
```

Comment:
This seems to have been introduced by another commit before in `main`, do we miss a previous PR to backport @Kyle-Verhoog?

Reply:
yeah we've got them all lined up now i think 😅